app/components/game-handler.hbs

```diff
@@ -0,0 +1,15 @@
+<div ...attributes data-test-game-handler class="game-handler">
+    <div data-test-sets-list class="sets-list">
+        {{#each @game.sets as |set index|}}
+            <SingleSet 
+                @set={{set}} 
+                @setNumber={{increment index}} 
+                @callback={{this.settleGame}} 
```

Comment:
I'm not a big fan of naming actions as callbacks. Maybe it would be better to just use an action name?  
`@onSettleGame={{this.settleGame}} `